dashboard/frontend/src/pages/Job.tsx

```diff
@@ -142,6 +153,38 @@ const JobPage: FC<{
     await handler()
   }, [])
```

Comment:
I've just realised that all my speil about including the `id` in the list of dependencies is dumb because this function also does not have that - so - let's apply the `id` to this list of dependencies also :-) i.e.

```ts
  const loadInfo = useCallback(async () => {
    const handler = loadingErrorHandler(async () => {
      const info = await api.get(`/api/v1/job/${id}/info`)
      setJobInfo(info)
    })
    await handler()
  }, [
    id,
  ])
```